src/app/design/design.component.ts

```diff
@@ -61,6 +61,7 @@ export class DesignComponent implements OnInit, OnDestroy {
             this.router.navigate([params['type'], 'design']);
           } else {
             // design was found so load it.
+            design.feature_type = (design.feature_type === 'hush') ? 'hush-block' : design.feature_type;
```

Comment:
I think you need to switch this. 
`design.feature_type = (design.feature_type === 'hush-block') ? 'hush' : design.feature_type;`